bin/tsv_to_vcf.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+import os
+import sys
+import re
+import argparse
+
+def parse_args(args=None):
+    Description = 'Reformat iVar tsv files to vcf files.'
+    Epilog = """Example usage: python tsv_to_vcf.py <FILE_IN> <FILE_OUT>"""
```

Comment:
```suggestion
    Epilog = """Example usage: python ivar_variants_to_vcf.py <FILE_IN> <FILE_OUT>"""
```

Reply:
I would rename the script because it is specifically for TSV files generated by IVar variants.